### ISSUE TITLE
Layout example for directly wired keys without matrix

### ISSUE THREAD

**Issue by Algram** [ahtn/keyplus#102]
Hi there,
first of all: This projects looks absolutely amazing. So good in fact that I started to design a pcb based on it for my gamepad.

In my version, every key is wired directly to a pin on the MCU. Here is a schematic:
![image](https://user-images.githubusercontent.com/5053729/71822848-d1680a80-308d-11ea-8376-05f757595168.png)

Does keyplus currently support a config like this, and if it does, could you maybe provide an example for me?

Thanks a lot!

**Comment by emilytrau** [comment]
The project hasn't been active for a while now, but it does support your configuration. Have a look at the Spectre PCB.

[PCB](https://github.com/ahtn/keyboard_pcb/tree/master/spectre)
[Layout](https://github.com/ahtn/keyplus/blob/master/layouts/spectre.yaml)

**Comment by Algram** [comment]
@angustrau very nice, so I could basically adapt the `layout.yaml` to my needs and use it in the keyplus firmware? I am using a different MCU than the specte but that does not seem to be a problem?

**Comment by emilytrau** [comment]
That's right. The 32a4u is a more or less equivalent MCU to the one in the spectre, and has pre-built firmware binaries available.

**Comment by Algram** [comment]
@angustrau perfect, I am very curious how my pad will turn out, the PCB and parts are already ordered. I am especially hoping for the lowest latency possible and I hope this can be achieved with keyplus.
